gocd/sp4.target.gocd.yaml

```diff
@@ -83,10 +85,16 @@ pipelines:
         - script: |-
             set -e
             PRODUCTS=$(osc -A https://api.suse.de ls SUSE:SLE-15-SP4:GA | grep "^SLES15-SP4" | grep -v release)
-            for product in kiwi-templates-Minimal sles15-image 000product $PRODUCTS; do
+            CONTAINERS=$(osc -A https://api.suse.de results -b -s succeeded SUSE:SLE-15-SP4:GA -r containerfile | cut -f1 -d ' '| uniq)
```

Comment:
I'm a bit suspicious of the `-s succeeded`. Images without that would silently be skipped. There could be a race between the check above and now...